docs/cli_reference.md

```diff
@@ -4170,7 +4170,7 @@ We advise to use Xen Orchestra instead of this method. See [Xen Orchestra rollin
 
 Commands for controlling VM scheduled snapshots and their attributes.
 
-The vmss objects can be listed with the standard object listing command (`xe vmss-list`), and the parameters manipulated with the standard parameter commands. For more information, see [Low-level parameter commands](#low-level-parameter-commands)
+The `vmss` objects can be listed with the standard object listing command (`xe vmss-list`), and the parameters manipulated with the standard parameter commands. For more information, see [Low-level parameter commands](#low-level-parameter-commands)
```

Comment:
Here's it's more a concept than a command, so either in caps or as an accepted word? By the way I'm all for adding accepted words for every concept that is met in the docs, rather than enforcing backticks each time we meet them. I don't think we'd lose much in quality and this would make contributions easier.

Reply:
I could be on board with that.  I think it's slightly more pleasing to the eye when these types of things are called out by the `backtick` style, but I agree with your point that not requiring them to be wrapped with backticks will lower contribution overhead.

Reply:
Should we add all XAPI classes as accepted words?

Reply:
Probably.